src/components/DropdownItem.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import { CheckIcon } from './icons';
+
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+
+const DropdownItem = ({
+  title,
+  description,
+  Icon,
+  className,
+  active,
+  onClick,
+  href,
+  target,
+  children,
+  disabled,
+  noItemsActiveState,
```

Comment:
I think it would be more clear if this prop had a name `showItemStatus` or something like this and be false by default